packages/riipen-ui/src/components/Button.test.jsx

```diff
@@ -0,0 +1,330 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTrash } from "@fortawesome/free-solid-svg-icons";
+
+import Button from "./Button";
+
+const icon = i => props => <FontAwesomeIcon icon={i} {...props} />;
+
+describe("<Button>", () => {
+  it("renders without errors", () => {
+    let error;
+
+    try {
+      mount(<Button />);
+    } catch (theError) {
+      error = theError;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const wrapper = mount(<Button />);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("default props", () => {
+    it("sets correct default props", () => {
+      const wrapper = mount(<Button value />);
+
+      expect(wrapper.find("Button").props().classes).toEqual([]);
+      expect(wrapper.find("Button").props().color).toEqual("default");
+      expect(wrapper.find("Button").props().disabled).toEqual(false);
+      expect(wrapper.find("Button").props().fullWidth).toEqual(false);
+      expect(wrapper.find("Button").props().size).toEqual("medium");
+      expect(wrapper.find("Button").props().type).toEqual("button");
+      expect(wrapper.find("Button").props().variant).toEqual("text");
+    });
+  });
+
+  describe("children prop", () => {
+    it("displays given children", () => {
+      const child = <div>child</div>;
+
+      const wrapper = mount(<Button>{child}</Button>);
+
+      expect(wrapper.containsMatchingElement(child)).toBeTruthy();
+    });
+  });
+
+  describe("classes prop", () => {
+    it("sets custom class", () => {
+      const classes = ["newClass"];
+
+      const wrapper = mount(<Button classes={classes} />);
+
+      expect(
+        wrapper
+          .find("Button")
+          .childAt(0)
+          .hasClass(classes[0])
+      ).toEqual(true);
+    });
+  });
+
+  describe("color prop", () => {
+    it("gives an error when given an invalid color", () => {
+      const errors = jest.spyOn(console, "error").mockImplementation();
+      const color = "blue";
+
+      mount(<Button color={color} />);
+
+      expect(errors).toHaveBeenCalledTimes(1);
+    });
+  });
+
+  describe("component prop", () => {
```

Comment:
We might be missing tests for this prop. We should be able to test Button.jsx on line 51 and 52:
```
let Component = component || other.href ? "a" : "button";
if (component) Component = component;
```